core/src/main/python/wlsdeploy/tool/util/targets/additional_output_helper.py

```diff
@@ -43,42 +42,46 @@
 WEBLOGIC_CREDENTIALS_SECRET = 'webLogicCredentialsSecret'
 
 
-def create_vz_configuration(model, model_context, aliases, exception_type):
+def create_additional_output(model, model_context, aliases, exception_type):
     """
-    Create and write the Kubernetes resource configuration files for Verrazzano.
-    :param model: Model object, used to derive some values in the configurations
-    :param model_context: used to determine location and content for the configurations
+    Create and write additional output for the configured target type.
+    :param model: Model object, used to derive some values in the output
+    :param model_context: used to determine location and content for the output
     :param aliases: used to derive secret names
     :param exception_type: the type of exception to throw if needed
     """
 
     # -output_dir argument was previously verified
     output_dir = model_context.get_kubernetes_output_dir()
```

Comment:
could model_context combine the two output_dir into one common routine?

Reply:
Done, combined into get_output_dir()